### ISSUE TITLE
can not access to the pre-processed coco2014 dataset

### ISSUE THREAD

**Issue by StevenLiuWen** [fmu2/3d_photo_stylization#1]
Hi, researchers,

Thanks for sharing this fantastic project. I want to train the model and find the link to the pre-processed coco2014 dataset (https://drive.google.com/drive/folders/1qtv-fBrCJXT93w0QoFbt8YqeI09pS8EM?usp=sharing) is broken (404 error) and also missing the files `test_ldi_render.py` and `test_ldi_model.py`.

Appreciate for repairing the broken link, and thanks in advance.

**Comment by xingyi-li** [comment]
`test_ldi_pcd.py` is also missing.

**Comment by xingyi-li** [comment]
Oh, I guess `test_ldi_render.py` is exactly `ldi_render.py`? Same for `test_ldi_model.py` and `test_ldi_pcd.py`.

**Comment by fmu2** [comment]
@xingyi-li you are correct. We will update README this weekend and fix the broken links.

**Comment by xingyi-li** [comment]
Hi @fmu2! Hope you are doing well. I understand you are busy with your research, but I am really interested in your work and hope you can update README and fix the broken links as soon as possible. One more thing, as mentioned at [issue #3](https://github.com/fmu2/3d_photo_stylization/issues/3), I am also curious about how to test the model on my own photo. So, could you provide a tutorial on this?

Many thanks!!!!
